docs/groovy-script/mods/mwc/crafting_station.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "MWC Crafting"
+titleTemplate: "Modern Warfare Cubed | CleanroomMC"
+description: "Converts multiple items into another item, with a way to recover some of original resources when dismantled. recipes with group ``BULLET, MAGAZINE, GRENADE`` can be only crafted via Ammo Press, while recipes with ``GUN, ATTACHMENT_NORMAL, ATTACHMENT_MODIFICATION`` categories can only be crafted with Crafting Station."
+source_code_link: "https://github.com/Cubed-Development/Modern-Warfare-Cubed/blob/next/src/main/java/com/paneedah/mwc/groovyscript/script/CraftingStation.java"
+---
+
+# MWC Crafting (Modern Warfare Cubed)
+
+## Description
+
+Converts multiple items into another item, with a way to recover some of original resources when dismantled. recipes with group ``BULLET, MAGAZINE, GRENADE`` can be only crafted via Ammo Press, while recipes with ``GUN, ATTACHMENT_NORMAL, ATTACHMENT_MODIFICATION`` categories can only be crafted with Crafting Station.
```

Comment:
why two backticks? only need one. might want to make it something like this:
> Any recipes with the group `BULLET`, `MAGAZINE`, or `GRENADE` can only be crafted via the Ammo Press
also, capitalization needs to be fixed. `dismantled. recipes` is incorrect.